components/org.wso2.carbon.identity.branding.preference.resolver/src/main/java/org/wso2/carbon/identity/branding/preference/resolver/UIBrandingPreferenceResolverImpl.java

```diff
@@ -666,6 +667,11 @@ private void clearCustomTextResolverCacheHierarchy(OrganizationManager organizat
 
     private void clearCustomTextResolverCache(String tenantDomain, String organizationId, String resourceName) {
```

Comment:
Better to have a separate test resolver cache clear method just passing the tenant domain and org id(if needed) but without a resourceName.

Then the idea is clear that, if someone wants to clear all the records in particular cache respective to that org need to invoke that method.

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/41/commits/aa0dfd886f8e0226f967d15dd07e3c4e49544a27